app/shared/src/commonMain/kotlin/ui/subject/episode/video/settings/DanmakuRegexFilterGroup.kt

```diff
@@ -53,60 +56,48 @@ internal fun isValidRegex(pattern: String): Boolean {
 
 @Composable
 internal fun SettingsScope.DanmakuRegexFilterGroup(
-    danmakuRegexFilterList: List<DanmakuRegexFilter>,
-    onAdd: (filter: DanmakuRegexFilter) -> Unit,
-    onEdit: (id: String, new: DanmakuRegexFilter) -> Unit,
-    onRemove: (filter: DanmakuRegexFilter) -> Unit,
-    onSwitch: (fiter: DanmakuRegexFilter) -> Unit,
+    danmakuRegexFilterState: DanmakuRegexFilterState,
```

Comment:
```suggestion
    state: DanmakuRegexFilterState,
```

对于 composable 的唯一一个 state 参数, 一般就叫做 `state` 以保证代码简短